### ISSUE TITLE
Support for venetian blinds

### ISSUE THREAD

**Issue by yolani** [resoai/TileBoard#700]
Not an issue but a feature request: Are there any plans to support venetian blinds in a single card? Would require. a cover card that can also take care of the tilt_position attribute, for example here:
<img width="362" alt="image" src="https://user-images.githubusercontent.com/219931/115690891-cd80fa00-a35d-11eb-914b-6dda3a9de2a4.png">

**Comment by alphasixtyfive** [comment]
How does this even look like in Lovelace?

**Comment by yolani** [comment]
You get 2 sliders, one for position and one tile_position:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/219931/115710896-2c9d3980-a373-11eb-9e0c-d3b72eeabfbd.png">

**Comment by yolani** [comment]
I tried to add sliders[] to my TYPES.COVER object but it does not seem to support them.

**Comment by alphasixtyfive** [comment]
No, sliders are only for light tile. You can simply add a second tile with a slider.
